pkg/flow/adapter/xslttransform/adapter.go

```diff
@@ -139,6 +142,23 @@ func (a *xsltTransformAdapter) dispatch(ctx context.Context, event cloudevents.E
 			fmt.Errorf("eror processing XML with XSLT: %v", err), nil)
 	}
 
+	if isXML {
```

Comment:
Is this whole new conditional related to the K_SINK story?

Reply:
yes sir.

Reply:
yes. because we have to know if we are sending back an XML or JSON payload right?

Reply:
Ah, that's for the response 👍